3-extra/3-fibonacci.js

```diff
@@ -14,8 +14,19 @@
 */
 
 function generateFibonacciSequence(n) {
-    // TODO
-}
+    if (n < 3) {
+        throw "Error";
+    }
+
+    let fibonacciSequence = [0, 1];
+
+    for (i = 2; i < n; i++) {
+        let fibonacciLastNumber = fibonacciSequence.length - 1;
```

Comment:
`fibonacciLastNumber` sounds like a value.
but this variable stands rather for the index for it.